diffpy/pdfmorph/pdfmorphapp.py

```diff
@@ -114,6 +116,7 @@ def createOptionParser():
     return parser
 
 def main():
+    plt.style.use(bg_mpl_style)
```

Comment:
Is this needed here, or only where it is used in the plotting module?  Delete here if it is not used (and delete the import)

Reply:
You're right, I saw some of the config settings and thought it might be used but didn't look deeper because the plot was working after changing the other file. Removed.